openverse_catalog/dags/providers/provider_api_scripts/freesound.py

```diff
@@ -179,7 +178,11 @@ def _get_audio_file_size(self, url):
 
         TODO(obulat): move filesize detection to the polite crawler
         """
-        return int(requests.head(url).headers["content-length"])
```

Comment:
It seems like the TODO task is done 😄 and we can remove the comment 👏
Aside from that, LGTM!

Reply:
Well not quite 😅 Since we don't have the polite crawler running at all. But I agree, we can remove the comment 🙂